packages/form-footer/src/FormFooter.story.tsx

```diff
@@ -41,8 +41,13 @@ const meta: StoryMetaType<typeof FormFooter> = {
     generate: {
       storyNames: ['LightMode', 'DarkMode'],
       combineArgs: {
-        backButtonText: [undefined, 'Back'],
-        cancelButtonText: ['', 'Cancel'],
+        backButtonProps: [
+          undefined,
+          { text: 'Back', leftGlyph: undefined },
+          { text: 'Back', leftGlyph: null },
```

Comment:
Do `leftGlyph: undefined` and `leftGlyph: null` render differently? If not, we can omit one of these. This is just for visual testing

Reply:
yes, was previously following the api described here: https://github.com/mongodb/leafygreen-ui/pull/2226#discussion_r1505127559

but upon further thought, I don't think I need to do that and will update